docs/config.json

```diff
@@ -1,17 +0,0 @@
-{
-  "title": "Crafty",
-  "tagline": "The task runner that binds them all",
-  "author": "Swissquote Bank SA",
-  "html": {
-    "theme": "swissquote",
-    "repo": "swissquote/crafty",
-    "google_analytics": "UA-113355094-2",
```

Comment:
not sure why you deleted the whole file, I think just removing this line was enough

Reply:
done